docs/source/man.rst

```diff
@@ -135,42 +131,42 @@ To create a credential, supply SSH credentials as either a username-password pai
 
 ``--sshpassphrase``
 
-  Prompts for the passphrase to be used when connecting using an ssh keyfile that requires a passphrase. Can only be used with the ``--sshkeyfile`` option.
-
-``--become-password``
-
-  Prompts for the privilege escalation password to be used when running a network scan.
+  Prompts for the passphrase to be used when connecting with an SSH keyfile that requires a passphrase. Can only be used with the ``--sshkeyfile`` option.
 
 ``--become-method=become_method``
 
-  Sets the method to become for privilege escalation when running an ansible playbook. Must be ``sudo``, ``su``, ``pbrun``, ``pfexec``, ``doas``, ``dzdo``, ``ksu``, ``runas``. The default is set to 'sudo' when the credential type is network.
+  Sets the method to become for privilege escalation when running an Ansible playbook. The value must be ``sudo``, ``su``, ``pbrun``, ``pfexec``, ``doas``, ``dzdo``, ``ksu``, or ``runas``. The default is set to ``sudo`` when the credential type is ``network``.
```

Comment:
I would just change this to:
_Sets the method to become for privilege escalation when running a network scan._

No need to mention Ansible.

Reply:
Fixed as suggested.